src/test/java/com/softserveinc/edu/ita/page_object/NewUserPage.java

```diff
@@ -0,0 +1,93 @@
+package com.softserveinc.edu.ita.page_object;
+
+
+import com.softserveinc.edu.ita.domains.User;
+import com.softserveinc.edu.ita.locators.NewUserPageLocators;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+
+/**
+ * PageObject Class presents page for new User creating
+ */
+public class NewUserPage extends LogOutBase {
+
+    public NewUserPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public AdministrationPage clickCreateButton() {
+        driver.findElement(NewUserPageLocators.CREATE_BUTTON.getBy()).click();
+        return new AdministrationPage(driver);
+    }
+
+    public void fillUserDataInput(By inputLocator, String inputValue) {
```

Comment:
Don't understand. If you mean clickCreateButton I can say that's old code. Now it's named
clickCreateUserLink